Gemfile

```diff
@@ -5,56 +5,54 @@ ruby '3.2.2'
 gem 'mime-types'
 
 # travis maintained gems
-gem 'travis-support',  git: 'https://github.com/travis-ci/travis-support'
+gem 'travis-support',  git: 'https://github.com/travis-ci/travis-support', ref: '4dda53ffa96b804db22c261551256caa18c4a2cc'
 gem 'travis-amqp',     git: 'https://github.com/travis-ci/travis-amqp'
 gem 'travis-config',  git: 'https://github.com/travis-ci/travis-config'
-gem 'travis-settings', git: 'https://github.com/travis-ci/travis-settings'
-gem 'travis-lock',     git: 'https://github.com/travis-ci/travis-lock'
-gem 'travis-github_apps', git: 'https://github.com/travis-ci/travis-github_apps'
+gem 'travis-settings', git: 'https://github.com/travis-ci/travis-settings', branch: 'master'
+gem 'travis-lock',     git: 'https://github.com/travis-ci/travis-lock/', branch: '6.1'
+gem 'travis-github_apps', git: 'https://github.com/travis-ci/travis-github_apps', branch: 'ga-ext_access'
 gem 'travis-rollout',  git: 'https://github.com/travis-ci/travis-rollout'
 gem 'simple_states',  git: 'https://github.com/travis-ci/simple_states', branch: 'prd-ruby-upgrade-dev'
-gem 'metriks',        git: 'https://github.com/travis-ci/metriks'
-gem 'metriks-librato_metrics', git: 'https://github.com/travis-ci/metriks-librato_metrics'
 gem 'marginalia', git: 'https://github.com/travis-ci/marginalia'
 gem 'gh',   git: 'https://github.com/travis-ci/gh'
 gem 'rollout',           git: 'https://github.com/travis-ci/rollout'
 
-gem 'mustermann'
-gem 'sinatra'
+gem 'sinatra', '~> 2'
 gem 'sinatra-contrib', require: nil #git: 'https://github.com/sinatra/sinatra-contrib', require: nil
 
 gem 'active_model_serializers' , '~> 0.9.9'
 gem 'unicorn'
 gem 'sentry-ruby'
 gem 'yard-sinatra',    git: 'https://github.com/rkh/yard-sinatra'
-gem 'rack', '~> 2.2'
-gem 'rack-contrib'
+gem 'rack-contrib', '= 2.3.0'
 gem 'rack-cache',      git: 'https://github.com/rtomayko/rack-cache'
 gem 'rack-attack', '~> 6'
 gem 'bunny',           '~> 2.22'
 gem 'dalli'
 gem 'pry'
+gem 'http', '~> 4'
+gem 'metriks',         '0.9.9.6'
+gem 'metriks-librato_metrics', git: 'https://github.com/eric/metriks-librato_metrics'
```

Comment:
keep the ones from master(metriks/metriks-librato_metrics

Reply:
Done